packages/xchain-mayachain-query/src/mayachain-query.ts

```
@@ -514,7 +517,7 @@ export class MayachainQuery {
       ? baseAmount(0, assetDecimals[assetToString(CacaoAsset)])
       : baseAmount(constantsDetails['TNSREGISTERFEE'], assetDecimals[assetToString(CacaoAsset)])
 
-    const totalFeePerBlock = baseAmount(constantsDetails['TNSFEEPERBLOCK']).times(
+    const totalFeePerBlock = baseAmount(constantsDetails['TNSFEEPERBLOCK'], assetDecimals[assetToString(CacaoAsset)]).times(
       numberOfBlocksToAddToExpiry > 0 ? numberOfBlocksToAddToExpiry : 0,
     )
```

ESLint

[error] Replace `constantsDetails['TNSFEEPERBLOCK'],·assetDecimals[assetToString(CacaoAsset)]).times(⏎······numberOfBlocksToAddToExpiry·>·0·?·numberOfBlocksToAddToExpiry·:·0,⏎····` with `⏎······constantsDetails['TNSFEEPERBLOCK'],⏎······assetDecimals[assetToString(CacaoAsset)],⏎····).times(numberOfBlocksToAddToExpiry·>·0·?·numberOfBlocksToAddToExpiry·:·0`

(prettier/prettier)